src/main/appengine/index.yaml

```diff
@@ -1,6 +1,18 @@
 # Reference: https://cloud.google.com/appengine/docs/standard/java11/configuring-datastore-indexes-with-index-yaml
 
 indexes:
+- kind: CourseStudent
+  properties:
+  - direction: asc
+    name: courseId
+  - direction: asc
+    name: email
+- kind: Instructor
+  properties:
+  - direction: asc
+    name: courseId
+  - direction: asc
+    name: email
```

Comment:
On a separate note, might need to investigate whether it's worth building the index for. Though rn it seems to be used in high impact data flow, but might not be all that necessary considering the scope it serves.

Reply:
I think Fergus mentioned an error would be thrown without the indices. Maybe when the staging server is set up, this can be checked.

Reply:
I understand. The indices are needed for the projection query as these queries read only from indexes. 
What I'm concerned about is the necessity of the index and whether we can simply remove projection to save on indexes. We will defer this to perhaps Wilson. It seems too great a cost (there are way more instructors and students than FeedbackSessions stored) for a rather niche scope.

Reply:
Oh I see, ok!